src/Fmt_odoc.ml

```diff
@@ -37,6 +37,10 @@ let fmt_ref_kind = function
   | RK_link -> str ":"
   | RK_custom s -> str s $ str ":"
 
+module List_kind = struct
+  type t = List | Enum
+end
```

Comment:
Nothing significant, but I'm not sure what the reasoning is behind adding a module vs just using a couple polymorphic variant tags?

Reply:
It's just a habit I have, I prefer to only use polymorphic variants when I need one of the specific features of polymorphic variants. I find that with regular variants you get errors closer to the actual location of the problem and nicer error messages.

I updated the code to use polymorphic variants instead.